src/MSBuild.Sdk.SqlProj/Sdk/Sdk.targets

```diff
@@ -46,22 +46,24 @@
       IncrementalClean;
       PostBuildEvent
     </CoreBuildDependsOn>
-    <GenerateNuspecDependsOn>$(GenerateNuspecDependsOn);SetNuSpecProperties</GenerateNuspecDependsOn>
+    <GenerateNuspecDependsOn>$(GenerateNuspecDependsOn);SetNuSpecProperties;_PackageDatabaseResults</GenerateNuspecDependsOn>
+    <PackageType Condition="'$(PackageType)'==''" >dacpac</PackageType>
```

Comment:
Hmm, I seem to remember that actually setting the package type breaks some things as you can no longer install these packages anymore. Regardless though we're not actually using this value currently.

Reply:
Without doing this some times the packages were being built incorrectly and didn't not have all of their children referenced correctly.